dnadoh/synthesize.py

```diff
@@ -196,10 +218,41 @@ def make_weight(self, person, reference, individual):
         mean = self.WEIGHT_MEANS[person.gsex]
         std_dev = mean * self.WEIGHT_RSD
         person.weight = _truncate(random.gauss(mean, std_dev), 1)
+    
+    def make_household_id(self, person, reference, individual):
+        person.household_id = self.hh_id
```

Comment:
so will this always be 0 to start? since `_init_household` always sets `self.hh_id = 0` ?